modal/__init__.py

```diff
@@ -99,3 +99,11 @@
     "web_server",
     "wsgi_app",
 ]
+
+
+def __getattr__(name):
```

Comment:
wow you can have a `__getattr__` on a module level 🤯

Reply:
will this also catch `from modal import Stub`?

Reply:
> will this also catch from modal import Stub?

unfortunately not:

```
$ python -c 'from modal import Stub'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ImportError: cannot import name 'Stub' from 'modal' (/home/ec2-user/client/modal/__init__.py)
```

Reply:
Another reason to avoid the `from x import y` spelling in documentation IMO.

Reply:
not super important but in theory we could keep the `Stub` class and turn the warning into an error. But I suspect this usage is uncommon